shimmingtoolbox/cli/sct_features.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python3
+
+import click
+import os
+import subprocess
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+
+@click.group(context_settings=CONTEXT_SETTINGS, help=f"Perform dilatation, erosion, deep segmentation and get "
+                                                     f"centerline of images.")
+def sct_features():
+    pass
+
+
+@sct_features.command(context_settings=CONTEXT_SETTINGS, help=f"Perform mathematical dilatations on images")
+@click.option('-input', 'fname_input', type=click.Path(), required=True, help="Input path of the nifti file")
+@click.option('-output', type=click.Path(), default=os.curdir, help="Output folder for mask in nifti file")
+@click.option('-dilate', type=int, required=True,
+              help="Dilate binary or greyscale image with specified size. If shape={'square', 'cube'}: size corresponds"
+                   "to the length of an edge (size=1 has no effect). If shape={'disk', 'ball'}: size corresponds to the"
+                   " radius, not including the center element (size=0 has no effect).")
+@click.option('-shape', type=click.Choice(['square', 'cube', 'disk', 'ball']), default='ball',
+              help="Shape of the structuring element for the mathematical morphology operation. Default: ball. If a 2D "
+                   "shape {'disk', 'square'} is selected, -dim must be specified. (default: ball)")
+@click.option('-dim', type=click.IntRange(0, 2),
+              help="Dimension of the array which 2D structural element will be orthogonal to. For example, if you wish "
+                   "to apply a 2D disk kernel in the X-Y plane, leaving Z unaffected, parameters will be: shape=disk, "
+                   "dim=2.")
+@click.option('-symmetrize', type=click.IntRange(0, 2), default=1, help="Symmetrize data along the specified "
+                                                                        "dimension.")
+@click.option('-verbose', type=click.IntRange(0, 2), default=1, help="Verbose: 0 = nothing, 1 = classic, 2 = expended.")
+def erode(fname_input, output, dilate, shape, dim, symmetrize, verbose):
```

Comment:
why is it sct-specific? if it's only an erosion it could probably go somewhere else

Reply:
I agree, I will change that. This branch will be more oriented towards the creation of an SCT mask and the get_centerline feature which is necessary to create a centerline process mask.